src/utils/getCapacity.ts

```diff
@@ -0,0 +1,24 @@
+import { find, round } from "lodash";
+import type { Maybe } from "../types";
+import type { DiskPartition } from "../services/lansweeper/types";
+
+const BYTE = 1;
+const KB = BYTE * 1024;
+const MB = KB * 1024;
+const GB = MB * 1024;
```

Comment:
These should be `KiB` if you want to us 1024 otherwise KB should be 1000. Probs a library for this.

Reply:
Fixed

Reply:
Maybe a product choice to let them decide between 1,000 and 1024 as base units.